app/views/groups/index.html.erb

```diff
@@ -17,17 +17,17 @@
           <tr>
             <td><%= group.title %></td>
             <td><%= group.location %></td>
-            <td><%= link_to 'Show', group %></td>
-            <td><%= link_to 'Edit', edit_group_path(group) %></td>
-            <td><%= link_to 'Destroy', group, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+            <td><%= link_to 'Show', group, class: "btn-show" %></td>
+            <td><%= link_to 'Edit', edit_group_path(group), class: "btn-edit" %></td>
+            <td><%= link_to 'Destroy', group, class:"btn-delete", method: :delete, data: { confirm: 'Are you sure?' } %></td>
           </tr>
         <% end %>
       </tbody>
     </table>
 
     <br>
 
-  <div class="row justify-content-center">
-    <%= link_to 'New Group', new_group_path, class: "btn btn-primary" %>
-  </div>
-</div>
+   
```

Comment:
[Code Hygiene]
Always check what you've committed. These empty lines could be removed next time.